docs/content/docs/connectors/table/jdbc.md

```diff
@@ -211,30 +211,48 @@ Connector Options
       <a href="https://jdbc.postgresql.org/documentation/head/query.html#query-with-cursor">Postgres</a>, may require this to be set to false in order to stream results.</td>
     </tr>
     <tr>
-      <td><h5>lookup.cache.max-rows</h5></td>
+      <td><h5>lookup.cache</h5></td>
+      <td>optional</td>
+      <td>yes</td>
+      <td style="word-wrap: break-word;">NONE</td>
+      <td><p>Enum</p>Possible values: NONE, PARTIAL</td>
```

Comment:
I think here we should list the three enum values, but explain only support the prior two strategies. WDYT?

Reply:
I'm afraid this could confuse users that we list an unavailable option in the doc, which increases the cost of explanation.